### PR TITLE
feat: add dynamic values for services, container ports, sidecars and add container tty/stdin

### PR DESCRIPTION
**Pull request by royki**
- Feature added
  - dynamic values for services, container ports
  - container stdin/tty
  - sidecars

### CODE REVIEW

**Review by KaamilW — APPROVED** [review]

File: values.yaml [hunk 1]
```diff
@@ -40,7 +40,9 @@ pvc:
   mountPath: "/tmp"
 
 
-containerPort: 80
+container:
```
Comment (michalziobro) [inline values.yaml:43]: @royki this is a breaking change, should be indicated by incrementing major sem ver

**Review by michalziobro — CHANGES_REQUESTED** [review]

**Comment by michalziobro** [comment]
@royki you need to bump version in Chart.yaml

File: Chart.yaml [hunk 2]
```diff
@@ -30,3 +30,9 @@ version: 0.1.3
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
 appVersion: "1.16.0"
+dependencies:
+  - name: common
+    repository: https://charts.bitnami.com/bitnami
+    tags:
+      - bitnami-common
+    version: 2.x.x
```
[the remark this answers is not in the document]
Comment (royki) [reply Chart.yaml]: marked `2.2.2`

File: values.yaml (hunk 1 above)
Comment (royki) [reply values.yaml:43]: bump chart version `1.0.0`

**Review by michalziobro — APPROVED** [review]

**Pull request merged**